connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignorTest.java

```diff
@@ -1508,25 +1545,25 @@ private List<Integer> allocations(Function<ConnectorsAndTasks, ? extends Collect
     private void assertNoRevocations() {
         returnedAssignments.newlyRevokedConnectors().forEach((worker, revocations) ->
                 assertEquals(
-                        "Expected no revocations to take place during this round, but connector revocations were issued for worker " + worker,
                         Collections.emptySet(),
-                        new HashSet<>(revocations)
+                        new HashSet<>(revocations),
+                        "Expected no revocations to take place during this round, but connector revocations were issued for worker " + worker
                 )
         );
         returnedAssignments.newlyRevokedTasks().forEach((worker, revocations) ->
                 assertEquals(
-                        "Expected no revocations to take place during this round, but task revocations were issued for worker " + worker,
                         Collections.emptySet(),
-                        new HashSet<>(revocations)
```

Comment:
it can be `assertEquals(0, revocations.size)` or `assertTrue(revocations.isEmpty)`

Reply:
We used `assertEquals` here in order to see the difference between expected and actual values if the assertion fails. Can we please keep that and not replace it with assertions about the size or emptiness of the collection instead?

Reply:
These tests aren't flaky, but it is a bit tricky to make changes to them. It definitely saved me time to have these error messages the last time I was in this neck of the woods.

With rudimentary checking logic (searching for `assertEquals(Collections.empty` in the code base), I also see this pattern used 94 times across 16 files in the Connect modules, so it's not exactly uncommon. IMO it'd be better to make it more common in other parts of the code base rather than try to "simplify" things in a way that drops information on assertion failures.

Reply:
@C0urante, thanks for your comments, I will rollback to use `assertEquals()`

Reply:
> With rudimentary checking logic (searching for assertEquals(Collections.empty in the code base), I also see this pattern used 94 times across 16 files in the Connect modules, so it's not exactly uncommon. IMO it'd be better to make it more common in other parts of the code base rather than try to "simplify" things in a way that drops information on assertion failures.

I file https://issues.apache.org/jira/browse/KAFKA-16929 to have more discussion